misc/skywater-pdk/build-lib.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+LIB=${PKG_NAME#skywater-pdk.}
+LIB_DIR=libraries/$LIB/v$PKG_VERSION
+mkdir -p $PREFIX/share/skywater-pdk/$LIB_DIR/
```

Comment:
we also need a `latest` symlink for `open_pdks` to be happy, see https://github.com/RTimothyEdwards/open_pdks/blob/dd84b77085a5bffba8d5493e82137322388997b9/sky130/Makefile.in#L918

Reply:
Done